src/modules/users/users.controller.ts

```diff
@@ -66,8 +66,6 @@ export class UsersController {
           const userDto: UserDto = new UserDto({
             auth0Id: userId,
             email: user.email,
-            name: user.nickname,
-            avatar: user.picture,
```

Comment:
`avatar` and `name` are required fields https://github.com/Coding-Coach/find-a-mentor-api/blob/master/src/modules/common/schemas/user.schema.ts#L34, if we try to save the record without this data, mongo will complain 🤔